### PR TITLE
Fix Elixir function placeholder text

### PR DESCRIPTION
**Pull request by lpil**
Elixir doesn't have methods, so I've updated this little snippet to be more accurate :)

### CODE REVIEW

File: elixir-mode/def [hunk 1]
```diff
@@ -2,6 +2,6 @@
 # name: def
 # key: def
 # --
-def ${1:method}${2:(${3:args})} do
+def ${1:function}${2:(${3:args})} do
```
Comment (AndreaCrotti) [inline elixir-mode/def]: if the "()" is always needed anyway I vote for @radik909 proposal.
If that's not the case leaving the option as it is now makes more sense imho..

Comment (lpil) [inline elixir-mode/def]: The `()` is not always needed. Convention is to not put `()` if it is a zero arity function.

Comment (radik909) [inline elixir-mode/def]: @lpil 
Got it!

**Comment by AndreaCrotti** [comment]
All good thanks

**Comment by lpil** [comment]
Thanks!

**Pull request merged**